frontend/src/scenes/events/definitions/EventPropertiesStats.tsx

```diff
@@ -0,0 +1,98 @@
+import { Input, Row, Table } from 'antd'
+import { useValues, useActions } from 'kea'
+import { ObjectTags } from 'lib/components/ObjectTags'
+import React, { useState } from 'react'
+import { useDebouncedCallback } from 'use-debounce/lib'
+import { definitionDrawerLogic } from './definitionDrawerLogic'
+
+export function EventPropertiesStats(): JSX.Element {
+    const { eventPropertiesDefinitions, eventsSnippet, eventPropertiesDefinitionTags, tagLoading } = useValues(
+        definitionDrawerLogic
+    )
+    const { setNewEventPropertyTag, deleteEventPropertyTag, setEventPropertyDescription } = useActions(
+        definitionDrawerLogic
+    )
+    const propertyExamples = eventsSnippet[0]?.properties
+    const tableColumns = [
+        {
+            title: 'Property',
+            key: 'property',
+            render: function renderProperty({ name }: { name: string }) {
```

Comment:
Reuse types as much as possible. This function could be `renderProperty({ name }: EventOrPropType)`, or maybe even `PropertyDefinition` or something. Same for all the other `renderStuff` below.